tests/test_table.py

```diff
@@ -300,14 +299,7 @@ def test_access():
                 pytest.DB['files'],
                 pytest.DB['segments'],
             ],
-            create_table(
-                audformat.utils.concat(
```

Comment:
Since the tables `pytest.DB['files']` and `pytest.DB['segments']` have references to schemes that change the dtype of the columns, but when we use `create_table()` we cannot change to those dtypes, since we cannot use schemes. So I changed to `+` here, though I'm not sure how meaningful the test is, since `+` is exactly what we want to test for here :)